test/test_Simulation.jl

```diff
@@ -28,3 +29,27 @@ end # testset
     pos = simulation.world.dogs.member[3].position
     @test all(pos .< Flase.getSheepCoords(simulation.world, pos))
 end
+
+
+world2 = World(
+    v0=1.0,
+    n_dogs=100,
+    boxsize=50.0,
+    motion=BrownianMotion(noise=0.5, friction=1.0),
+    sheeps=DenseSheeps(20, n_sheeps=20),
+)
+
+simulation2 = Flase.ClusterTimeSimulation(;
+    condition=0,
+    dt=0.2,
+    world=world2,
+    plotter=Flase.VoidPlotter()
+)
+
+@testset "ClusterTimeSimulation" begin
+    mqd = Flase.MQD(simulation2.world.sheeps)
+    msd = Flase.MSD()
+    @test Flase.getMSD(msd, simulation2.world.sheeps, simulation2.world) < 0.7
```

Comment:
```suggestion
    @test Flase.getMSD(msd, simulation2.world.sheeps, simulation2.world) >= 0.7
```
Having a wrong test passing is worse than having a correct test failing